airflow/docker.go

```diff
@@ -85,8 +85,8 @@ func generateConfig(projectName, airflowHome string) string {
 	return buff.String()
 }
 
-// createProjectFromContext creates project with yaml config as context
-func createProjectFromContext(projectName, airflowHome string) (project.APIProject, error) {
+// fetchDockerProject creates project with yaml config as context
+func fetchDockerProject(projectName, airflowHome string) (project.APIProject, error) {
```

Comment:
still not sure about the naming of this func

Reply:
Yea, how about just `createProject`?